server/src/main/java/org/opensearch/common/unit/Fuzziness.java

```diff
@@ -234,9 +239,9 @@ private int termLen(String text) {
 
     public String asString() {
         if (isAutoWithCustomValues()) {
-            return fuzziness.toString() + ":" + lowDistance + "," + highDistance;
+            return fuzziness + ":" + lowDistance + "," + highDistance;
         }
-        return fuzziness.toString();
+        return fuzziness;
```

Comment:
why removing toString()?

Reply:
`fuzziness` itself is [declared as a string](https://github.com/opensearch-project/OpenSearch/blob/ac3351cc54e6fa1934fe0c15f3d1e3cdbef153b7/server/src/main/java/org/opensearch/common/unit/Fuzziness.java#L65), so `.toString()` is redundant